### ISSUE TITLE
Add git-lfs

### ISSUE THREAD

**Issue by viceice** [renovatebot/docker-buildpack#139]
* download linux-amd64 from [releases](https://github.com/git-lfs/git-lfs/releases/latest)
* extract
* copy binary to path `/usr/local/bin` (alternate is using the included `install.sh`)
* run `git lfs install` as `ubuntu` user

blocks: https://github.com/renovatebot/renovate/issues/6842

We should use [other](https://github.com/git-lfs/git-lfs/wiki/Installation#other) install method, as its faster (install) that the apt version. We should also use this because the available version on ubuntu is old.

**Comment by fgblomqvist** [comment]
We're hitting this issue, so figured I might as well tackle it. Any comments/thoughts other than what was stated in the original post before I do it?

**Comment by viceice** [comment]
See new issue

**Comment by viceice** [comment]
Duplicate of containerbase/buildpack#13
